mealie/services/scraper/cleaner.py

```diff
@@ -95,15 +96,20 @@ def clean_image(image: str | list | dict | None = None, default="no image") -> s
     if not image:
         return default
 
-    match image:  # noqa - match statement not supported
+    match image:
         case str(image):
             return image
+        case {"url": str(url)}:
+            return url
         case list(image):
-            return image[0]
-        case {"url": str(image)}:
```

Comment:
Is it correct for this case to be completely removed though?

Reply:
I just moved it slightly down so it's easier to read, it's not gone